packages/documentation-framework/scripts/webpack/webpack.base.config.js

```diff
@@ -55,20 +55,12 @@ module.exports = (_env, argv) => {
             options: {
               cacheDirectory: '.cache/babel',
               cacheCompression: false,
-              presets: [['@babel/preset-env', {
-                loose: true,
-                corejs: 3,
-                useBuiltIns: 'entry',
```

Comment:
I've removed the `core-js` auto-polyfilling and related settings here. The reason for that is with `useBuiltIns` set to `entry` it is required to add an explicit import to `core-js` in the application code ([see documentation](https://babeljs.io/docs/babel-preset-env#usebuiltins-entry)).

Since the aforementioned import never existed in the application code this essentially means that `core-js` polyfills were also never included. So I have opted to plain out disable it to make this more explicit.